charts/seed-managedresource/templates/managedresource.yaml

```diff
@@ -0,0 +1,22 @@
+---
```

Comment:
Rather than having a template for this, can't we have functions that create the secrets + the managed resource?